platform_umbrella/apps/verify/lib/verify/application.ex

```diff
@@ -13,8 +13,7 @@ defmodule Verify.Application do
       Verify.SessionURLAgent,
       {Task.Supervisor, [name: Verify.TaskSupervisor]},
       {CommonCore.Installs.Generator, [name: Verify.Installs.Generator]},
-      {Verify.KindInstallWorker, [name: Verify.KindInstallWorker]},
-      # for BatteryInstall & ImagePull workers
+      # for workers
       {Registry, keys: :unique, name: Verify.Registry}
```

Comment:
We're starting a new worker per test module now. That'll help with state mgmt and cleanup especially as we'll have multiple clusters per module.